export.js

```diff
@@ -135,6 +135,14 @@ Avatar = {
   // Get the url of the user's avatar
   getUrl: function (user) {
 
+    // This is reactive, which causes Avatar.getUrl to trigger reactivity when
+    // it is used in reactive computations. There's no harm done to code that
+    // uses this function in non-reactive places.
+    var _user = Meteor.user()
```

Comment:
This code will cause a re-render for every change to the Meteor.user() object even if there's another user specified for the Avatar. That's pointless. Instead you should simply do:

if (!user) user = Meteor.user()

This also avoids the _ prefix which doesn't follow the coding style of the rest of the code base :)